install/kubernetes/helm/istio/charts/grafana/templates/deployment.yaml

```diff
@@ -136,3 +142,8 @@ spec:
         configMap:
           name:  istio-grafana-configuration-dashboards-{{ $filename }}
 {{- end }}
+{{- range $key, $value := .Values.customDashboards }}
+      - name: dashboards-custom-{{ $key | replace "." "_" }}
+        configMap:
+          name:  custom-grafana-configuration-dashboards-{{ $key | replace "." "_" }}
```

Comment:
Does this configmap really exist? Looks like we only have configmap named `istio-grafana-configuration-dashboards-{{ $filename }}`.

Reply:
Fixed